nilearn/connectome/group_sparse_cov.py

```diff
@@ -957,13 +966,21 @@ def fit(self, subjects, y=None):
             alpha_1, _ = compute_alpha_max(emp_covs, n_samples)
             alpha_0 = 1e-2 * alpha_1
             alphas = np.logspace(np.log10(alpha_0), np.log10(alpha_1),
-                               n_alphas)[::-1]
+                                 n_alphas)[::-1]
 
         covs_init = itertools.repeat(None)
+
+        # Copying the cv generators to use them n_refinements times.
+        cv_list = []
+        cv_ = izip(*cv)
+        for i in range(n_refinements):
+            _, cv_ = itertools.tee(cv_)
+            cv_list.append(cv_)
```

Comment:
Just this comment to address before the merge. To be clear, what I have in mind is something like:

</pre>
n_refinements = 3

cv = (i**2 for i in range(5))

import itertools

# Use itertools.tee to duplicate multiple times the cv iterator
for this_cv in itertools.tee(cv, n_refinements):
    print(list(this_cv))
</pre>

And then you can use the cv object just as before.